cli/azd/pkg/infra/provisioning/provider.go

```diff
@@ -21,10 +21,12 @@ type Options struct {
 	Provider ProviderKind `yaml:"provider"`
 	Path     string       `yaml:"path"`
 	Module   string       `yaml:"module"`
+	BicepADS bool
 }
 
 type DeployResult struct {
-	Deployment *Deployment
+	Deployment                  *Deployment
+	SameAsLastDeploymentSkipped bool
```

Comment:
This name is a bit confusing.  I think the intent is to return a value letting the command know whether or not the provisioning was skipped because it is the same as the last deployment?  Maybe `Skipped` is enough? If we need more we could introduce a `SkipReason` attribute?